src/content/docs/distributed-tracing/troubleshooting/infinite-tracing-trace-configuration-conflicts.mdx

```diff
@@ -21,6 +21,9 @@ A tracing configuration conflict can cause issues like:
 * Incomplete Infinite Tracing metrics due to not taking standard trace sampling into account.
 * Discrepancies for metrics like span count, service count, durations, and error count.
 * Confusing search results. For example, for standard tracing spans that send headers to Infinite Tracing-instrumented services, those spans may show in the trace list but not in the trace waterfall view.
-* Missing traces for monitored browser apps and mobile apps. Because Infinite Tracing isn’t yet available for browser or mobile monitoring, spans from these services won’t show up in the trace waterfall when they make requests to Infinite Tracing-enabled services.
+* Missing traces for monitored browser apps and mobile apps. Because Infinite Tracing isn't yet available for browser or mobile monitoring, spans from these services won't show up in the trace waterfall when they make requests to Infinite Tracing-enabled services.
+
+For traces with a configuration conflict, we'll display only the Infinite Tracing data in the UI because that data is of higher quality. (All tracing data is queryable via NRQL.)
+
+You can use our [instrumentation status app](https://one.newrelic.com/dt-setup-assistant) to see an overview of your trace instrumentation. To find this, go to: **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Apps > Distributed tracing instrumentation status**.
```

Comment:
@zuluecho9 I don't know where "Apps" is anymore in the UI. I'm not seeing this under "All capabilities" either.